docs/developer/testing/TESTING.md

```diff
@@ -119,7 +119,22 @@ E2E tests are defined to run in the CI, so to run the locally, you may need to c
 - the related PersistentVolumeClaims to your provider in `tests/e2e/sample-applications/mysql-persistent/pvc-twoVol/`, `tests/e2e/sample-applications/mysql-persistent/pvc/` and `tests/e2e/sample-applications/mongo-persistent/pvc/` folders with your storage classes (to list cluster drivers, run `oc get storageclasses`)
 - Optionally, the user can use the default storage class by choosing the pvc/default_sc.yaml files.
 
-If running E2E tests against operator created from `make deploy-olm`, remember its image expires, which may cause **Subscription Config Suite Test** to fail.
+If running E2E tests against operator created from `make deploy-olm`, remember its image expires, which may cause tests to fail.
+
+When running Virtual Machine backup/restore tests on IBM Cloud, it is better to manually install OpenShift Virtualization operator, instead of automatically installing it through `make test-e2e`. Because this may cause the error of Virtual Machines never starting to run. Example test log:
```

Comment:
Is it fixable? Let's open an issue?

Reply:
the virt test checks if virt operator is installed https://github.com/openshift/oadp-operator/blob/master/tests/e2e/virt_backup_restore_suite_test.go#L142 

if yes, continues; otherwise, installs and configure it for you

Reply:
It could also patch the StorageProfile automatically, but for now I guess manual setup is the only way to make it work. I can work on this in another pull request.